src/client/pages/Debugger/ActualDebugger.tsx

```diff
@@ -4,13 +4,18 @@ import { colorsV3 } from '@hedviginsurance/brand'
 import { SessionContainer } from 'containers/SessionContainer'
 import { Button } from 'components/buttons'
 import { StorageContainer } from 'utils/StorageContainer'
+import { Label } from 'components/inputs/index'
 import { QuoteData } from './components/QuoteData'
 import { Session } from './components/Session'
 
 const Wrapper = styled.div`
   max-width: 900px;
   margin: 0 auto;
   color: ${colorsV3.gray100};
+
+  ${Label} {
```

Comment:
I guess declaring a dependency with a component just for styling purposes like that can be a little bit hard to follow since `Label` is not explicitly present in `ActualDebugger` scope (`ActualDebugger` > `OfferData` > `InputField` > `Label`). But I don't know how to reach the same result in a different way, so...

I guess the problem is that `body` `background` is set to black in globalStyles so everything inherits that color. So maybe in this case the more suitable solution would be change Debuggers background. Any thoughts?!

Either way, I'm ok with this solution :)

Reply:
When implementing this change I initially thought it would be easier to just change the Label. 
I agree it is not the best approach, however I feel like this will be the only exception that will be made to change the Label color. 
Also, worth mentioning that the Debugger is used internally so we can change the background/ label colors how we feel like it whenever we need to.